packages/shared/src/constants.tsx

```diff
@@ -21,6 +25,24 @@ export const SOCIAL_MEDIA_ICON_MAPPING: Record<string, ReactNode> = {
     [OPENSEA_ID]: <OpenSeaColoredIcon />,
 }
 
+export const SOCIAL_MEDIA_ROUND_ICON_MAPPING: Record<string, ReactNode> = {
+    [TWITTER_ID]: <TwitterRoundIcon />,
```

Comment:
Use `EnhanceableSite` instead.

Reply:
I will replace these ID to EnhanceableSite at other pr